src/Compilers/Core/Portable/WellKnownTypes.cs

```diff
@@ -252,10 +252,10 @@ internal enum WellKnownType
         System_ValueTuple,
         System_ValueTuple_T1,
         System_ValueTuple_T2,
-        System_ValueTuple_T3,
 
         ExtSentinel, // Not a real type, just a marker for types above 255 and strictly below 512
 
+        System_ValueTuple_T3,
```

Comment:
> Didn't follow why this change was made. Can you elaborate?

```ExtSentinel``` must have value 255. Since ```First``` has changed (```First = SpecialType.Count + 1,```), ```System_ValueTuple_T3``` should change value from 254 to 256. The build will be broken overwise.